src/cljs/athens/electron/dialogs.cljs

```diff
@@ -0,0 +1,96 @@
+            new-dir          (.resolve path new-dir base-dir-name)
+            new-dir-images   (.resolve path new-dir utils/IMAGES-DIR-NAME)
+            new-db-filepath  (.resolve path new-dir utils/DB-INDEX)]
+        (if (.existsSync fs new-dir)
+          (js/alert (str "Directory " new-dir " already exists, sorry."))
+          (do (.mkdirSync fs new-dir)
+              (.copyFileSync fs curr-db-filepath new-db-filepath)
+              (rf/dispatch [:db-picker/move-db curr-db-filepath new-db-filepath])
+              (rf/dispatch [:db/update-filepath new-db-filepath])
+              (when (.existsSync fs curr-dir-images)
+                (.mkdirSync fs new-dir-images)
+                (let [imgs (->> (.readdirSync fs curr-dir-images)
+                                array-seq
+                                (map (fn [x]
+                                       [(.join path curr-dir-images x)
+                                        (.join path new-dir-images x)])))]
+                  (doseq [[curr new] imgs]
+                    (.copyFileSync fs curr new))))))))))
+
+
+(defn open-dialog!
+  "Allow user to open db elsewhere from filesystem."
+  []
+  (let [res       (.showOpenDialogSync dialog (clj->js {:properties ["openFile"]
+                                                        :filters    [{:name "Transit" :extensions ["transit"]}]}))
+        open-file (first res)]
+    (when (and open-file (.existsSync fs open-file))
+      (rf/dispatch-sync [:db-picker/add-new-db open-file])
+      (rf/dispatch-sync [:db-picker/select-new-db open-file @(rf/subscribe [:db/synced])]))))
+
+
+;; mkdir db-location/name/
+;; mkdir db-location/name/images
+;; write db-location/name/index.transit
+(defn create-dialog!
+  "Create a new database."
+  [db-name]
+  #_(let [res         (.showOpenDialogSync dialog (clj->js {:properties ["openDirectory"]}))
+          db-location (first res)]
+      (when (and db-location (not-empty db-name))
+        (let [db          (d/empty-db db/schema)
+              dir         (.resolve path db-location db-name)
+              dir-images  (.resolve path dir utils/IMAGES-DIR-NAME)
+              db-filepath (.resolve path dir utils/DB-INDEX)]
+          (if (.existsSync fs dir)
+            (js/alert (str "Directory " dir " already exists, sorry."))
+            (do
+              (rf/dispatch-sync [:init-rfdb])
+              (rf/dispatch [:local-storage/create-db-picker-list])
+              (.mkdirSync fs dir)
+              (.mkdirSync fs dir-images)
+              (.writeFileSync fs db-filepath (dt/write-transit-str db))
+              (rf/dispatch [:fs/watch db-filepath])
+              (rf/dispatch [:db/update-filepath db-filepath])
+              (rf/dispatch [:db-picker/add-new-db db-filepath])
+              (rf/dispatch [:reset-conn db])
+              (rf/dispatch [:transact athens-datoms/datoms])
+              (rf/dispatch [:loading/unset])))))))
```

Comment:
Much of this code can be refactored and reused in `:fs/create-and-write`. Basically parameterize and pass in a directory rather than using the default directory (`~/Documents/athens`).